docs/pages/getting-started/overview.md

```diff
@@ -11,18 +11,18 @@ As the cloud native management plane, Meshery enables the adoption, operation, a
 
 Meshery features can be categorized by:
 
-1. Service Mesh Performance Management
+1. Cloud Native Performance Management
    - Workload and service mesh performance characterization
    - Prometheus and Grafana integration
-1. Service Mesh Configuration Management
+1. Cloud Native Configuration Management
    - Configuration best practices
-1. Service Mesh Lifecycle Management
+1. Cloud Native Lifecycle Management
    - Service mesh provisioning and workload onboarding
    - Meshery Operator and MeshSync
    - Service mesh patterns and Open Application Model integration
```

Comment:
We're missing a couple of references that can be swapped out here...

Reply:
Those were sub-points, hence I left them earlier, have updated them now.